CONTRIBUTING.md

```diff
@@ -248,7 +249,7 @@ Every new source file must begin with a license header.
 Most of Dgraph, Badger, and the Dgraph clients (dgo, dgraph-js, pydgraph and dgraph4j) are licensed under the Apache 2.0 license:
 
     /*
-     * Copyright 2016-2018 Dgraph Labs, Inc. and Contributors
+     * Copyright 2016-2022 Dgraph Labs, Inc. and Contributors
```

Comment:
The year is almost over, it would be good to put 2023.

Reply:
Not hard to do in 2023 ;)